Include/internal/pycore_runtime.h

```diff
@@ -83,6 +83,7 @@ typedef struct pyruntimestate {
            always have an ID of 0.  Overflow results in a RuntimeError.
            If that becomes a problem later then we can adjust, e.g. by
            using a Python int. */
+        int allow_new;
```

Comment:
It looks like this got inserted in the wrong place, considering that the comment above applies to the field below. 😄

Reply:
gah, will fix that